src/ai/backend/agent/server.py

```diff
@@ -556,6 +556,28 @@ async def commit(
             "path": str(Path(subdir, filename)),
         }
 
+    @rpc_function
+    @collect_error
+    async def get_local_config(self):
+        log.debug("rpc::get_local_config()")
```

Comment:
Why don't you erase it because it comes out too often? Because of this, it is difficult to check other logs.

Reply:
okay. seems this log is not useful that much